src/components/pool-display.tsx

```
@@ -3,6 +3,10 @@ import { useReadContract } from "wagmi";
 import { stakingContract } from "../constants/contracts";
 import { useErc20Token } from "../hooks/erc20Token";
 import { useState } from "react";
+import { waitForTransactionReceipt } from "viem/actions";
+import { useStatusMessage } from "../context/status-message.tsx";
+import { useToast } from "../ui/toast";
+import { Button } from "./button.tsx";
 import { Button } from "./button";
```

Biome (2.3.13)

[error] Shouldn't redeclare 'Button'. Consider to delete it or rename it.

'Button' is defined here:


(lint/suspicious/noRedeclare)